HPXMLtoOpenStudio/measure.rb

```diff
@@ -3652,10 +3653,17 @@ def self.get_space_temperature_schedule(model, location, spaces)
 
     sch = OpenStudio::Model::ScheduleConstant.new(model)
     sch.setName(location)
+    heating_setpoint = @hpxml.hvac_controls[0].heating_setpoint_temp
```

Comment:
Seems like this variable is now unused, right?

Reply:
Oops, I thought I deleted it. Good catch.